packages/cli/commands/functions/deploy.js

```diff
@@ -60,6 +61,26 @@ const loadAndValidateOptions = async options => {
   }
 };
 
+const logBuildOutput = async resp => {
+  return new Promise((resolve, reject) => {
+    try {
+      https
+        .get(resp.cdnUrl, response => {
+          let data = '';
+          response.on('data', chunk => {
+            data += chunk;
+          });
+          response.on('end', () => {
```

Comment:
Would it be worth it to handle 404s in case the log is not available?

Reply:
Good call, I added handling for this.